spreadsheets/kind/pois.csv

```diff
@@ -97,3 +97,10 @@ station,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,${13},station
 *,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,bed_and_breakfast;chalet;guest_house;hostel,*,${17},${tourism}
 *,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,subway_entrance,*,*,*,*,*,*,*,*,*,*,*,${17},subway_entrance
 *,*,bench;waste_basket,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,${18},${amenity}
+*,*,*,*,*,*,*,*,*,*,*,beacon;cross;mineshaft,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,${15},${man_made}
+*,*,*,*,*,*,*,*,*,*,*,adit,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,${16},adit
+*,*,*,*,*,*,*,*,*,*,*,water_well,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,${17},water_well
+*,*,*,*,*,*,*,*,*,*,*,*,saddle,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,${14},saddle
+*,*,*,*,*,*,*,*,*,*,*,*,dune;geyser;sinkhole,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,${15},"${""natural""}"
```

Comment:
We have to quote the column name `natural` as `"natural"` because natural is a keyword in SQL, and the double-quotes means it can be used as a column or table name. The `${}` around it is our way of quoting literals in the CSV generation, and the expansion into `"${""natural""}"` comes from the CSV itself: because it contains a quote, it must be quoted, and the original quotes are expanded to two consecutive sets each.